store/items.js

```diff
@@ -59,8 +60,11 @@ export const getters = {
   counts(state) {
     return state.counts
   },
+  timerange(state) {
+    return state.timerange
+  },
   deadline(state) {
```

Comment:
you can pass getters as second argument `deadline(state, getters)` see Vuex documentation https://vuex.vuejs.org/guide/modules.html#module-local-state